Documentation/get-dotnetcore-dnx-linux.md

```diff
@@ -0,0 +1,127 @@
+Get the .NET Core DNX SDK on Linux
+==================================
+
+These instructions will lead you through acquiring the .NET Core DNX SDK via the [.NET Version Manager (DNVM)](https://github.com/aspnet/dnvm)  and running a "Hello World" demo on Linux. The instructions use a particular set of paths. You'll need to adjust if you want to use a different set.
+
+These instructions are for .NET Core console apps. If you want to try out ASP.NET 5 on top of .NET Core - which is a great idea - check out the [ASP.NET 5 instructions](https://github.com/aspnet/home).
+
+.NET Core NuGet packages and the .NET Core DNX SDKs are available on the [ASP.NET 'vnext' myget feed](https://www.myget.org/F/aspnetvnext), which you can more easily view on [gallery](https://www.myget.org/gallery/aspnetvnext) for the feed.
+
+You can also [build from source](linux-instructions.md). 
+
+Environment
+===========
+
+These instructions are written assuming the Ubuntu 14.04 LTS, since that's the distro the team uses. Pull Requests are welcome to address other environments as long as they don't break the ability to use Ubuntu 14.04 LTS.
+
+Packages
+--------
+
+Install the following packages: 
+
+- libssl-dev
+- libunwind8 
+- unzip
+
+	sudo apt-get install libunwind8 libssl-dev unzip
+
+You also need a latest version of Mono, which is required for DNU. This is a temporary requirement. 
+
+	sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
+	echo "deb http://download.mono-project.com/repo/debian wheezy main" | sudo tee /etc/apt/sources.list.d/mono-xamarin.list
+	sudo apt-get update
+	sudo apt-get install Mono-Complete
+
+Certificates
+------------
+
+You need to import trusted root certificates in order to restore NuGet packages. You can do that with the `mozroots` tool.
+
+	mozroots --import --sync
```

Comment:
This isn't required anymore since Mono 3.12.0. Certificates are automatically imported when Mono is installed. I discovered recently that some systems like Linux Mint are configured in a way that this doesn't happen automatically, in those cases make sure the `ca-certificates-mono` package is installed.